chart/values.yaml

```diff
@@ -624,6 +659,11 @@ cleanup:
   # Run every 15 minutes
   schedule: "*/15 * * * *"
 
+  # Select certain nodes for airflow cleanup pods.
+  nodeSelector: {}
+  affinity: {}
+  tolerations: []
```

Comment:
values table in Readme does not contain this values for cleanup, redis, pgbouncer and other pods

Reply:
added.